### PR TITLE
Add outreach page

### PR DESCRIPTION
**Pull request by glass-ships**
Addresses #395

### CODE REVIEW

File: frontend/src/pages/about/PageOutreach.vue [hunk 1]
```diff
@@ -6,60 +6,108 @@
   <AppSection>
     <AppHeading>Outreach</AppHeading>
     <p>
-      The Monarch Initiative is a community effort, and we welcome your participation!
+      The Monarch Initiative is a community effort, and we welcome your
```
Comment (vincerubinetti) [inline frontend/src/pages/about/PageOutreach.vue:9]: The section component isn't intended to be nested. You should close the first section here then start a new one.

File: frontend/src/pages/about/PageOutreach.vue [hunk 2]
```diff
@@ -6,60 +6,108 @@
...
+      participation!
     </p>
     <AppSection>
       <AppHeading>Ways to Get Involved</AppHeading>
 
       <ul>
         <li>
-          <a href="https://oboacademy.github.io/obook/courses/monarch-obo-training/" target="_blank">OBO Academy: Monarch Training Series</a>
+          <a
```
Comment (vincerubinetti) [inline frontend/src/pages/about/PageOutreach.vue]: You should use the AppLink component here. This allows all links across the site to have consistent behavior, such as having a little arrow icon for external links like this.

Might also take this opportunity to check the other about pages for `<a>`'s that I may have missed.

File: frontend/src/pages/about/PageOutreach.vue [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+        <AppButton
+          text="Monarch LinkedIn Profile"
+          to="https://www.linkedin.com/company/the-monarch-initiative/"
+        />
+        <AppButton text="Monarch Blog" to="https://monarchinit.medium.com/" />
+        <AppButton
+          text="Monarch YouTube Channel"
+          to="https://www.youtube.com/@monarchinitiative"
+        />
+      </AppFlex>
```
Comment (vincerubinetti) [inline frontend/src/pages/about/PageOutreach.vue]: I'd recommend adding some icons to these buttons to draw more users eyes to them.

File: frontend/src/pages/about/PageOutreach.vue [hunk 4]
```diff
@@ -0,0 +1,146 @@
...
+    </AppSection>
+    <AppSection>
+      <AppHeading>For questions or technical support</AppHeading>
+      <p>
+        We welcome all questions, requests, and feedback! Please submit your
```
Comment (vincerubinetti) [inline frontend/src/pages/about/PageOutreach.vue]: Is this section necessary? Perhaps we could move some of this text to the Help page (where I'd argue it belongs), and then here just have a big button link to it.

Comment (sagehrke) [reply frontend/src/pages/about/PageOutreach.vue]: yes. agree.

File: frontend/src/pages/about/PageOutreach.vue (hunk 3 above)
Comment (sagehrke) [reply frontend/src/pages/about/PageOutreach.vue]: yes. agree. please add the logos for each type of website we are linking to. (youtube, linkedin, github, medium)

File: frontend/src/pages/about/PageOutreach.vue [hunk 5]
```diff
@@ -0,0 +1,146 @@
...
+            <li>
+              Youtube channel:
+              <a href="https://www.youtube.com/@obo-academy" target="_blank"
+                >https://www.youtube.com/@obo-academy</a
+              >
+            </li>
+            <li>
+              Join the OBO Academy mailing list using
+              <a href="https://groups.google.com/g/obo-academy" target="_blank"
+                >https://groups.google.com/g/obo-academy</a
```
Comment (sagehrke) [inline frontend/src/pages/about/PageOutreach.vue]: Embed these links to match the others and change the wording to to say:
> Join the OBO Academy mailing list [here](https://groups.google.com/g/obo-academy)

File: frontend/src/pages/about/PageOutreach.vue [hunk 6]
```diff
@@ -0,0 +1,146 @@
...
+              user needs.
+            </li>
+            <li>
+              These calls are scheduled for the second Thursday of every month.
+              You can find more information about the calls
+              <a
+                href="https://mondo.monarchinitiative.org/pages/workshop/#outreach"
+                target="_blank"
+                >here</a
+              >.
```
Comment (sagehrke) [inline frontend/src/pages/about/PageOutreach.vue]: Please make the additional information about the Mondo Outreach calls sub-bullets to the original description.

File: frontend/src/pages/about/PageOutreach.vue [hunk 7]
```diff
@@ -0,0 +1,146 @@
...
+            </li>
+            <li>
+              You can find the schedule with Zoom information
+              <a
+                href="https://docs.google.com/document/d/1MStpDyh9LOZYJTjLtnpOsNYc3HaeU-bz0CHAI9xOjfQ/edit#heading=h.6sqkx1xhumse"
+                target="_blank"
+                >here</a
+              >.
```
Comment (sagehrke) [inline frontend/src/pages/about/PageOutreach.vue]: Add:

> Join the Mondo User mailing list [here](https://groups.google.com/g/mondo-users)

File: frontend/src/pages/about/PageOutreach.vue [hunk 8]
```diff
@@ -0,0 +1,146 @@
...
+          >
+          <p>
+            LinkML is a flexible modeling language that allows you to author
+            schemas in YAML that describe the structure of your data.
+          </p>
+          <p>
+            LinkML is used to model the Monarch data model, and we welcome your
+            contributions to the LinkML community. You can learn more
+            <a
+              href="https://linkml.io/linkml/faq/contributing.html"
```
Comment (sagehrke) [inline frontend/src/pages/about/PageOutreach.vue]: Make this a bullet point

> You can learn more [here](https://linkml.io/linkml/faq/contributing.html)

File: frontend/src/pages/about/PageOutreach.vue (hunk 3 above)
Comment (sagehrke) [reply frontend/src/pages/about/PageOutreach.vue]: I'd also remove the word "Monarch" from each button. Just have the name of the platform with the emblem on each button.

File: frontend/src/pages/about/PageOutreach.vue [hunk 9]
```diff
@@ -0,0 +1,146 @@
...
+        </li>
+      </ul>
+    </AppSection>
+    <AppSection>
+      <AppHeading>Follow us to learn more!</AppHeading>
+      <p>
+        Stay updated on the latest from the Monarch Initiative by joining our
+        <a
+          href="https://groups.google.com/a/tislab.org/g/monarch-users"
+          target="_blank"
```
Comment (sagehrke) [inline frontend/src/pages/about/PageOutreach.vue]: Please remove this sentence and link. We are discussing the best way to go about this mailing list and it is not ready for prime time.

**Review by vincerubinetti — APPROVED** [review]

**Comment by glass-ships** [comment]
@nlharris just wanted to check in with you before merging this - are there any other changes/additions you'd like to see on this page?

**Comment by nlharris** [comment]
TBH this is hard to assess in this format -- I can look it over once it's merged into the website and make any suggestions then, ok? Please go ahead and merge.

**Comment by glass-ships** [comment]
Sounds good, thanks. 

If you'd like, you can see it rendered through the netlify preview here:
https://deploy-preview-397--monarch-app.netlify.app/outreach

But we'll go ahead and merge for now

**Pull request merged**